build-caching/src/main/java/com/vertispan/j2cl/build/PropertyTrackingConfig.java

```diff
@@ -213,6 +213,14 @@ public String getLanguageOut() {
     @Override
     public Path getWebappDirectory() {
         // Note that this deliberately circumvents the hash building
-        return Paths.get(config.findNode("webappDirectory").readString());
+        ConfigValueProvider.ConfigNode configNode=config.findNode("webappDirectory");
+        if(configNode==null) {
+            throw new IllegalStateException("No 'webappDirectory' found");
+        }
+        String s = configNode.readString();
+        if(s==null) {
```

Comment:
whitespace nits, after `if`, around `==`

Reply:
What style checker is at work there? :-)

Reply:
Absolutely a fair argument to make, but it might take longer to get your code merged.

Hey, want to make a patch to add a code formatter?

(Consistency is more important to me than "i like the formatter", but I've been very unhappy with "very consistent" formatters that take intent away, like chained stream calls each being on their own line, etc.)

Reply:
I stopped having any opinions on code formatting :-) Consistency is really good to have better merges. I was just curious if you use some kind of tooling for automatic formatting which I could maybe replicate. **Manually copying the existing style is also fine.** I was just not paying attention, as I was used to some auto-formatting by IDE anyway. At work we use the google code formatter but have issues with latest Java 17 constructs. Nothing is perfect.